src/main/java/com/xebialabs/overthere/winrs/WinrsConnection.java

```diff
@@ -102,7 +102,7 @@ public OverthereProcess startProcess(final CmdLine cmd) {
         winrsCmd.addArgument("winrs");
         winrsCmd.addArgument("-remote:" + address + ":" + port);
         winrsCmd.addArgument("-username:" + username);
-        winrsCmd.addPassword("-password:" + password);
+        winrsCmd.addRaw("-password:" + password);
```

Comment:
How will this work when the password contains other non-safe characters such as spaces, colons (':'), etc? Isn't the answer to improve the encoding? Or to add another type of encoding?

Reply:
I guess the only non-safe character is space. The other punctuation is not escaped anyway (https://github.com/xebialabs/overthere/blob/master/src/main/java/com/xebialabs/overthere/util/WindowsCommandLineArgsSanitizer.java#L27)
The main problem is that escaping in particular % sign escaping breaks authentication.
I guess quoting password should solve the problem, anyway it's the way we do it in case of whitespaces https://github.com/xebialabs/overthere/blob/master/src/main/java/com/xebialabs/overthere/util/WindowsCommandLineArgsSanitizer.java#L41

Reply:
Actually, it's a good comment. It has to be changed.